### PR TITLE
Adds nevrean eggs.

### LINKED ISSUE #1751: Nevrean need their egg type selectable
#### Brief description of the issue
Nevrean don't have an egg type selectable even though their egg is in the game and has a sprite

#### What you expected to happen
Egg be there

#### What actually happened
Egg not be there

#### Steps to reproduce
- use eyes

#### Code Revision
- release - 
256e1db4d6e43368e3e53cf3027e5ae395abdacd 
 
#### Anything else you may wish to add:
- Note to self: Add to 
https://github.com/VOREStation/VOREStation/blob/5ccb9395318b4f0b477008ac121cc10e78aee32a/code/_helpers/global_lists_vr.dm#L90-L112

- and
https://github.com/VOREStation/VOREStation/blob/5c7f6f98635fa0814e0e566cc12d419907a8455d/code/modules/client/preference_setup/vore/03_egg.dm#L1-L10

- And egg is here
https://github.com/VOREStation/VOREStation/blob/93888221b214c6a6d52743204791502dc230eeef/code/game/objects/structures/crates_lockers/closets/egg_vr.dm#L30

### PR DESCRIPTION
**Pull request by lbnesquik**
Fixes #1751

### CODE REVIEW

File: code/_helpers/global_lists_vr.dm [hunk 1]
```diff
@@ -106,6 +106,7 @@ var/global/list/global_egg_types = list(
 		"Tajaran" 		= TAJARAN_EGG,
 		"Akula" 		= AKULA_EGG,
 		"Skrell" 		= SKRELL_EGG,
+		"nevrean"		= NEVREAL_EGG
```
Comment (Screemonster) [inline code/_helpers/global_lists_vr.dm]: "nevreal"?

**Comment by lbnesquik** [comment]
Active map edges again.

**Comment by KasparoVy** [comment]
Noice.

**Comment by Arokha** [comment]
Don't spam commits to try and fix the AE thing, just ask a maintainer to rerun travis if you think your code is fine but travis is not

**Pull request merged**